test/data/azure/get_certs_for_model.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Curl + openssl has issues reading the system certificates on Android.
+# Pending a better solution we create an in-memory certificate store from certificates included in the model.
+#
+# The certificates must be added to the first Azure operator in the model in an attribute called 'x509_certificates'.
```

Comment:
seems like picking the first one could be tricky. e.g., if there are multiple Azure ops with no dependencies on each other, is one guaranteed to be first?

but pending a better solution, it seems fine for now. if we keep this approach later, I think it may be better to specify the global cert configuration at a higher level (e.g., in an environment variable) rather than as an op attribute.

Reply:
if we need to support multiple sets of certs, we would need to make them an explicit input to the node as well as having a per-node cert store. not sure if that is ever likely though. 

otherwise I don't think the order matters. one azure op needs the certs, and cert store init happens during kernel creation (which is during session state finalization and prior to any execution). 

but ideally we can find a solution where we can use the system certs so they're not adding to model size.

Reply:
oh, I see that the op just uses a non-empty cert attribute to initialize the store.